### DISCUSSION TITLE
Help with a non supported platform

### DISCUSSION THREAD

**Discussion by devinesteve** [artisan-roaster-scope/artisan#953] [Q&A] [answered]
I don't know if any of you have seen this device (https://www.inovato.com/) Its Linux based ( Debian Bullseye 64-bit ) computer
I hoped to get it running artisan with a small (8 inch) display - so I don't have to crowd my laptop onto my already crowded roasting table.

uname -a : Linux inovato 5.15.59-sunxi64 #trunk SMP PREEMPT Tue Aug 9 16:45:59 PDT 2022 aarch64 GNU/Linux

I have downloaded and installed the artisan-linux-2.6.0.deb package.
When I run it I get several errors like:
"Error: Symbol posix_spawn_file_actions_addopen not found, cannot apply R_X86_64_JUMP_SLOT @0xffff8bca9178 (0x5f2f0) in /usr/share/artisan/libpython3.10.so.1.0
Error: Symbol posix_spawnattr_setschedparam not found, cannot apply R_X86_64_JUMP_SLOT @0xffff8bca9510 (0x5fa20) in /usr/share/artisan/libpython3.10.so.1.0
Error: Symbol sched_setparam not found, cannot apply R_X86_64_JUMP_SLOT @0xffff8bca9ec0 (0x60d80) in /usr/share/artisan/libpython3.10.so.1.0"
It seems as if I am missing a library that contains spawn.h but it also seems as if artisan comes with all required libs
installed in /usr/share/artisan. 
Any thoughts on this?

**Comment by mrpenner** [comment, marked as answer]
Those errors mention X86_64. I think the artisan-linux-2.6.0.deb is x86 specific. Your device appears to be arm64 based. There is a build for raspberry pi, but that might be only 32-bit? I'm not sure how difficult it would be to build an arm64 version.

**Reply by devinesteve** [reply]
Yes, of course. I was looking right at. Thanks, I'll try the raspberry pi package. I've been trying to build from source but that's been failing too. I will update this later

**Reply by devinesteve** [reply]
Just for the record. I gave up trying to get the Artisan software to run on this platform - moving on to a Raspberry Pi.
